nrf_security/cmake/kconfig_mbedtls_configure.cmake

```diff
@@ -213,45 +218,81 @@ configure_file(${CMAKE_CURRENT_LIST_DIR}/../configs/nrf-config.h.template
                ${CMAKE_CURRENT_BINARY_DIR}/include/nrf-config-noglue.h)
 
 #
-# Enabling ALT functionality
-# This includes GLUE, CC310 or other ALT backends
+# Generate glue header file for nrf_oberon (with oberon specific alt flags
+# temporary set)
+#
+if (CONFIG_OBERON_BACKEND)
+  # Temporary set the AES setkey and encrypt/decrypt
+  set(MBEDTLS_AES_SETKEY_ENC_ALT TRUE)
+  set(MBEDTLS_AES_SETKEY_DEC_ALT TRUE)
+  set(MBEDTLS_AES_ENCRYPT_ALT TRUE)
+  set(MBEDTLS_AES_DECRYPT_ALT TRUE)
+  set(MBEDTLS_ECDH_GEN_PUBLIC_ALT TRUE)
+  set(MBEDTLS_ECDH_COMPUTE_SHARED_ALT TRUE)
+  set(MBEDTLS_ECDSA_GENKEY_ALT TRUE)
+  set(MBEDTLS_ECDSA_SIGN_ALT TRUE)
+  set(MBEDTLS_ECDSA_VERIFY_ALT TRUE)
+  set(MBEDTLS_ECJPAKE_ALT TRUE)
+  set(MBEDTLS_SHA1_ALT TRUE)
+  set(MBEDTLS_SHA256_ALT TRUE)
+
+  configure_file(${CMAKE_CURRENT_LIST_DIR}/../configs/nrf-config.h.template
+                 ${CMAKE_CURRENT_BINARY_DIR}/include/nrf-config-noglue-oberon.h)
+
+  # Reset the defines again
+  set(MBEDTLS_AES_SETKEY_ENC_ALT FALSE)
+  set(MBEDTLS_AES_SETKEY_DEC_ALT FALSE)
+  set(MBEDTLS_AES_ENCRYPT_ALT FALSE)
+  set(MBEDTLS_AES_DECRYPT_ALT FALSE)
```

Comment:
Why isn't `MBEDTLS_ECP_ALT` here?

Reply:
Because the oberon does not replace ECC on ECP-layer only using the listed mechanisms